lib/Memory/Memorable.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <EEPROMex.h>
+#include <EEPROMVar.h>
+
+const int START = 0;
+const int SIZE = 32;
+
+class Memorable {
+public:
+  virtual void read();
```

Comment:
If it's a pure virtual class you should write method like this `virtual void read () = 0;` This make that you cannot create this class. You can only inheritance from it and it force inheriting class to write a virtual method in it.
Same to `write()` function.